src/LibHdb++MySQL.h

```diff
@@ -154,19 +185,20 @@ class HdbPPMySQL : public AbstractDB
 	MYSQL *dbp;
 	unsigned long db_mti;
 	string m_dbname;
-	map<string,int> attr_ID_map;
+	unordered_map<string,int> attr_ID_map;
 	bool lightschema;	//without recv_time and insert_time
 	bool jsonarray;
 	bool autodetectschema;
 	bool ignoreduplicates;	//ignore duplicated key (att_conf_id,data_time) insert failures
-	map<string,int> attr_ERR_ID_map;
+	uint32_t batch_size{DEFAULT_BATCH_SIZE};
```

Comment:
I don't think it makes much of a difference but why uint32_t instead of unsigned int ?

Reply:
> I don't think it makes much of a difference but why uint32_t instead of unsigned int ?

No reason. There are some others around. I'll check.